test/llvm/PREDIFF

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+TESTNAME=$1
+OUTFILE=$2
+
+TMPFILE="$outfile.prediff.tmp"
+
+FILECHECK=${CHPL_HOME}/third-party/llvm/install/linux64-gnu/bin/FileCheck
+
+mv $OUTFILE $TMPFILE
+$FILECHECK --input-file $TMPFILE $TESTNAME.chpl 2> $OUTFILE
+rm $TMPFILE
```

Comment:
Can you put this script it `util/test` and then symbolically link to it, in the event we need to use it in more places? I'd still like to see `FileCheck` as a 1st class alternative to a `.good` file in the testing system but until then, let's use a symbolic link to a shared implementation of this file.